test/it/client_it.js

```diff
@@ -16,19 +16,16 @@ describe('Client', function() {
   describe('creation', function() {
     it('should not throw', function(done) {
       assert.doesNotThrow(function () {
-        helpers.loadApiKey(function(apiKey) {
-          var client = new stormpath.Client({
-            apiKey: apiKey,
-            skipRemoteConfig: true
-          });
-
-          client.on('error', function (err) {
-            throw err;
-          });
-
-          client.on('ready', function () {
-            done();
-          });
+        var client = new stormpath.Client({
+          skipRemoteConfig: true
+        });
+
+        client.on('error', function (err) {
+          throw err;
+        });
+
+        client.on('ready', function () {
```

Comment:
Why? In this case throwing an exception makes more sense. Consider this, the client readies, then when calling something, a client error happens. And by doing what you suggest it's not certain it will be handled, since the `done()` callback is just expected to be called once. Whereas any exception being thrown will be picked up properly by mocha.